components/Navigation/NavigationLogin.vue

```diff
@@ -0,0 +1,34 @@
+<template>
+  <ClientOnly>
+    <ul style="list-style: none;">
+      <li v-if="loginStatus">
+        <nuxt-link to="/user/logout">Logout</nuxt-link>
```

Comment:
it will have to toggle / refresh the loginstatus  after doing so. does it do it correctly? if not, let's simply use a regular a link to force-refresh the whole page? simple and effective? a like-wise simple vue-based solution would be fine also of course.